1621763679085	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1621764279087	Sync.Service	DEBUG	User-Agent: Waterfox/78.10.0 (Linux x86_64) FxSync/1.80.0.20210415101645.desktop
1621764279087	Sync.Service	INFO	Starting sync at 2021-05-23 13:04:39 in browser session 05_udG3Qe-yy
1621764279088	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1621764279096	Sync.Status	INFO	Resetting Status.
1621764279096	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1621764279100	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1621764279100	FirefoxAccounts	DEBUG	getOAuthToken enter
1621764279103	FirefoxAccounts	TRACE	getCachedToken returning cached token
1621764279103	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1621764279105	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1621764279336	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/155119483/info/collections failed: [Exception... "<no message>"  nsresult: "0x805a2ffe (<unknown>)"  location: "<unknown>"  data: no] No traceback available
1621764279337	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "<no message>"  nsresult: "0x805a2ffe (<unknown>)"  location: "<unknown>"  data: no] No traceback available
1621764279339	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1621764279350	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "<no message>"  nsresult: "0x805a2ffe (<unknown>)"  location: "<unknown>"  data: no] No traceback available
